src/ansys/fluent/core/fluent_connection.py

```diff
@@ -285,6 +362,87 @@ def id(self) -> str:
         """Return the session id."""
         return self._id
 
+    def force_exit(self):
```

Comment:
I think that, regardless of server-side changes, it is a good idea to have this functionality in PyFluent, as it can also deal with situations where the server/container is unresponsive or frozen.